view_tasker_project_summary.py

```
@@ -0,0 +1,201 @@
+    print()
+
+    for f in summary_files:
+        mtime = datetime.fromtimestamp(f.stat().st_mtime)
+        size = f.stat().st_size
+
+        # Count lines (exclude header)
+        try:
+            with open(f, 'r') as file:
+                line_count = sum(1 for line in file if not line.startswith('#'))
+                line_count_str = str(line_count)
+        except:
```

Ruff (0.14.0)

Do not use bare `except`

(E722)